### ISSUE TITLE
Add common searches to the home page.

### ISSUE THREAD

**Issue by bruno-garcia** [NuGetTrends/nuget-trends#93]
Often comparison of logging libraries or web frameworks happen.

The home page could have some links to those to help people landing on the site for the first time.

**Comment by joaopgrassi** [comment]
Are these based on frequent searches or just some "hard-coded" examples? We could start with hard-coded ones and then extend later to something smarter.

**Comment by bruno-garcia** [comment]
That's true. Hard coded works fine I believe, at first at least. I'm always interested in logging for example: https://nugt.net/s/gecctk9
